pom.xml

```diff
@@ -78,5 +78,30 @@
         <maven.compiler.release>8</maven.compiler.release>
       </properties>
     </profile>
+    <profile>
```

Comment:
@costin-zaharia-sonarsource is this really needed if the parent pom is updated?
I looked at e,g, sonar-php whcih got released, it doesn't have this

Reply:
this is already in the parent pom https://github.com/SonarSource/parent-oss/blob/e7e21714eee9a861d2371b174f7ab804f7a79f06/pom.xml#L577

Please rebase on master and remove this

Reply:
The agent runs as a `NetworkService` and for this user we don't have a default `.m2` folder from where the script can take the signing key.

In order to fix this we need to override the profile and set the path to the key file.